### PR TITLE
feat(mini): added missing actions of about teritori 

### PR DESCRIPTION
**Pull request by ChiragPansuriya-iView**
Added link on below ss 
simulator_screenshot_83C07B43-A150-4526-9180-347FCB01CED2

### CODE REVIEW

File: packages/screens/Mini/About/AboutScreen.tsx [hunk 1]
```diff
@@ -22,27 +27,27 @@ type AboutScreenProps = {
...
   },
   {
     label: "Help & Support",
-    url: "https:ggogle.com",
+    url: "https://teritori.notion.site/Privacy-Policy-16e2332744d346db9b78909a91cb44e3",
   },
   {
     label: "GitHub",
-    url: "https:ggogle.com",
+    url: "https://github.com/TERITORI",
```
Comment (omniwired) [inline packages/screens/Mini/About/AboutScreen.tsx:42]: https://github.com/TERITORI/teritori-dapp 🙏

File: packages/screens/Mini/About/AboutScreen.tsx [hunk 2]
```diff
@@ -22,27 +27,27 @@ type AboutScreenProps = {
...
   },
   {
     label: "Terms of Use",
-    url: "https:ggogle.com",
+    url: "https://teritori.notion.site/Terms-Conditions-6007160a0c74472cbb75a07bdfdd5f73",
   },
   {
     label: "Help & Support",
-    url: "https:ggogle.com",
+    url: "https://teritori.notion.site/Privacy-Policy-16e2332744d346db9b78909a91cb44e3",
```
Comment (omniwired) [inline packages/screens/Mini/About/AboutScreen.tsx:38]: I think the best one we have is discord!

**Review by omniwired — APPROVED** [review]
✨🚀

**Review by n0izn0iz — APPROVED** [review]
LGTM

**Pull request merged**